packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -159,16 +179,19 @@ export async function pollDeploymentStatus(deployCtx: DeployContext) {
             currentStatus[operation.resourceName] = operation.status;
             if (!polledOperations.includes(operation.resourceName)) {
               polledOperations.push(operation.resourceName);
-              const subOperations = await deployCtx.client.deploymentOperations.list(
-                deployCtx.resourceGroupName,
-                operation.resourceName
-              );
-              subOperations.forEach((sub) => {
-                const subOperation = getRequiredOperation(sub, deployCtx);
-                if (subOperation) {
-                  currentStatus[subOperation.resourceName] = subOperation.status;
-                }
-              });
+
+              if (operation.subscriptionId === deployCtx.client.subscriptionId) {
```

Comment:
we may need to handle cases that subscriptionId is different?

Reply:
It is fixed. Thanks.